pdr_backend/sim/dash_plots/view_elements.py

```diff
@@ -73,3 +84,60 @@ def selected_var_checklist(state_options, selected_vars_old):
         id="selected_vars",
         style={"display": "none"},
     )
+
+
+def get_tabs_component(elements):
+    return dcc.Tabs(
```

Comment:
we could also use dbc Tabs for consistency with the lake dash

Reply:
I took a look now at dbc Tabs and I'll prefer to use dcc because is offers more flexibility.
An impactful example is that dbc doesn't offer the option to display the tabs vertically on the left side.